add description for trusted scriptlets (#51)
AdguardTeam/KnowledgeBase: docs/general/ad-filtering/create-own-filters.md

```
This rule will be applied to `example.org` and subdomains pages and will execute the `abort-on-property-read` scriptlet with the `alert` parameter.

Learn more about [how to debug scriptlets](#debug-scriptlets).

More information about scriptlets can be found [on GitHub](https://github.com/AdguardTeam/Scriptlets#scriptlets).

> **Compatibility with different versions of AdGuard**
>
> Scriptlet rules are not supported by AdGuard Content Blocker.

### Trusted scriptlets {#trusted-scriptlets}

Trusted scriptlets are [scriptlets](#scriptlets) with extended functionality. Their names are prefixed with `trusted-`, e.g `trusted-set-cookie`, to be easily distinguished from common scriptlets.

> **Restrictions**
>

> > Trusted scriptlets rules can be used **only in trusted filters**. This category includes:

Reviewer: Но это ведь справедливо и для обычных скриптлетов или в расширении по-другому?

Author: справедливо, не по-другому

Reviewer: тогда будет логичнее сделать эту заметку где-нибудь [здесь](https://github.com/AdguardTeam/KnowledgeBase/pull/51/files#diff-190ba525f7c4d0b0eda33dab9df98df01d3ac072d6aeef951f044c96d58a25c9R2082)?

Author: вообще да. ещё про эти трастед-фильтры много где упоминается и каждый расписано что это значит, резонно описать в одном месте и только ссылаться потом

>
> Scriptlet rules are not supported by AdGuard Content Blocker.

### Trusted scriptlets {#trusted-scriptlets}

> Trusted scriptlets are [scriptlets](#scriptlets) with extended functionality. Their names are prefixed with `trusted-`, e.g `trusted-set-cookie`, to be easily distinguished from common scriptlets.

Reviewer 2: e.g.

> **Restrictions**
>
> Trusted scriptlets rules can be used **only in trusted filters**. This category includes:

> > * filter lists [created by AdGuard Team](./adguard-filters.md),

Reviewer 2: by the AdGuard team

> * custom filter lists installed as `trusted`,
> * user rules.

> **Note**
>

> > Trusted scriptlets are not compatible with any other blocker.

Reviewer 2: Trusted scriptlets are not compatible with other ad blockers except AdGuard.

```
This rule will be applied to `example.org` and subdomains pages and will execute the `abort-on-property-read` scriptlet with the `alert` parameter.

Learn more about [how to debug scriptlets](#debug-scriptlets).

More information about scriptlets can be found [on GitHub](https://github.com/AdguardTeam/Scriptlets#scriptlets).

> **Compatibility with different versions of AdGuard**
>
> Scriptlet rules are not supported by AdGuard Content Blocker.

### Trusted scriptlets {#trusted-scriptlets}

Trusted scriptlets are [scriptlets](#scriptlets) with extended functionality. It means the same syntax and restrictions. Trusted scriptlet names are prefixed with `trusted-`, e.g. `trusted-set-cookie`, to be easily distinguished from common scriptlets.

> **Note**
>
> Trusted scriptlets are not compatible with other ad blockers except AdGuard

> **Compatibility with different versions of AdGuard**
>

> > Trusted scriptlets rules are supported now by AdGuard for Windows, Mac, and Android, **running CoreLibs version 1.10.141 or later**.

Reviewer 2: Now trusted scriptlets rules are supported или Trusted scriptlets rules are now supported

ExtendedCss.query(selectorText) // returns an array of elements matching selectorText
```

### Debugging scriptlets {#debug-scriptlets}

> If you are using AdGuard Browser extension and want to debug a [scriptlet](#scriptlets) or [trusted scriptlet](#trusted-scriptlets) rule, you can get additional information by simply having the Filtering log opened. In that case, scriptlets will switch to debug mode and will write more information to the browser console.

Reviewer 2: ```suggestion
If you are using AdGuard Browser extension and want to debug a [scriptlet](#scriptlets) or a [trusted scriptlet](#trusted-scriptlets) rule, you can get additional information by opening the Filtering log. In that case, scriptlets will switch to debug mode and there will be more information in the browser console.
```
